### DISCUSSION TITLE
How to implement "wakeup" words in coqui-stt?

### DISCUSSION THREAD

**Discussion by BurhanMuhyiddin** [coqui-ai/STT#2256] [Q&A] [answered]
Hi all. I have implemented a Rasa chatbot via rasa, bert, coqui-stt and tts. However, I want to add the functionality of "wakeup" word for the chatbot, so it will not listen and convert text to speech every time, but only commanded (by wake up command). Is there possibility of implementing this via coqui-stt? If yes, how can I do that? Thank you very much for your time.

**Comment by wasertech** [comment, marked as answer]
You need a small yes/no type scorer with only your wake-word(s) [see [relevant doc](https://stt.readthedocs.io/en/latest/LANGUAGE_MODEL.html)]. You'll also most likely have to play with negative and positive values for hotword boosting to make sure only your wake words are picked up [see [relevant doc](https://stt.readthedocs.io/en/latest/HotWordBoosting-Examples.html)].
Then it's just a matter of unloading the wakeup scorer and loading another one.

**Reply by BurhanMuhyiddin** [reply]
Thank you very much for your answer. I will read the links you have wrote.
